[WIP]feat(rbac): turn rbac plugin into a dynamic plugin

ciiay opened the pull request:
For [RHIDP-1050](https://issues.redhat.com/browse/RHIDP-1050): Add/enable RBAC UI plugin to showcase/RHDH and test the RBAC UI flow

invincibleJai commented:
/cc @davidfestal 
/cc @PatAKnight

divyanshiGupta commented on .turbo-cookie:
This file is not needed

divyanshiGupta commented on packages/backend/src/plugins/kubernetes.ts:
Looks like this file got pushed accidentally.

divyanshiGupta commented on yarn.lock:
Looks like while rebasing some conflict headers/separators were not removed. I will suggest that you discard these changes and pull yarn.lock again from main and then do `yarn install` to correctly generate any changes and avoid any issues that might occur because of a bad yarn.lock file.

divyanshiGupta commented on plugins/rbac-backend/dist-dynamic/package.json:
Do we need this file?

ciiay replied:
No, the entire `plugins/rbac-backend/dist-dynamic/` is not needed. Added accidentally. It's been removed.

divyanshiGupta commented on plugins/rbac-backend/dist-dynamic/yarn.lock:
Same as above. Do we need this?

davidfestal commented:
/approve

ciiay commented:
@divyanshiGupta Thanks for the review. I addressed the comments you left and removed changes in `yarn.lock` as I didn't change anything in `package.json`. Please confirm this updated PR, thanks.

invincibleJai commented:
/lgtm
